jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ConfigFilesFacade.kt

```diff
@@ -188,6 +193,26 @@ class DefaultConfigFilesFacade(
         }
     }
 
+    override fun deleteSsoConnectionFromConfig(sessionName: String) {
```

Comment:
could we refactor this and the previous method since it's largely the same logic

Reply:
While the initial logic of locating the section is the same, I think it's cleaner to keep the update and delete logic separate. It's easier to debug and change later....

Reply:
it is more difficult to change if you have to remember to make the change in two different places